common/src/main/java/org/wso2/testgrid/common/InfraCombination.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.testgrid.common;
+
+/**
+ * This defines s single infra combination.
+ *
+ * @since 1.0.0
+ */
+public class InfraCombination {
+    private String os;
```

Comment:
We already have an InfrastructureCombination in place. We need to reuse existing stuff!

Reply:
Yes, we have an InfrastructureCombination but in order to generate the graph, we used a simplified version. It is used to maintain OS, JDK and DB Engine.

Reply:
Then this needs to go to reporting module, not to common. This'll lead to
major confusion otherwise.

Reply:
We are using the same object for mapping the values in mustache, so shall we move this into report models?